lms/templates/student_account/login_and_register.html

```diff
@@ -22,6 +22,7 @@
             newrelic.addPageAction('xfinished');
         }
     </%static:require_module>
+    <script type="text/javascript" src="${static.url('js/student_account/tos_modal.js')}"></script>
```

Comment:
Awesome, thanks! SiteConfiguration is cached in memory for 5 minutes. So you can either restart or wait the 5 minutes.